pkg/kubectl/cmd/util/helpers.go

```diff
@@ -591,28 +589,30 @@ func ParsePairs(pairArgs []string, pairType string, supportRemove bool) (newPair
 		removePairs = []string{}
 	}
 	var invalidBuf bytes.Buffer
-
+	var invalidBufNonEmpty bool
 	for _, pairArg := range pairArgs {
-		if strings.Index(pairArg, "=") != -1 {
+		if strings.Contains(pairArg, "=") {
```

Comment:
@AdoHe @smarterclayton its simpler and have better readability, as suggested by https://github.com/dominikh/go-simple